conf/locale

```diff
@@ -0,0 +1 @@
+/home/salah/edx-proctoring/edx_proctoring/locale
```

Comment:
This is wrong, it should be a relative path instead of having your name in it.

```
$ cd edx-proctoring/conf
$ rm -f locale
$ ln -sd ../edx_proctoring/locale locale
```